Gems/AWSGameLift/cdk/aws_gamelift/flex_match/flex_match_configurations.py

```diff
@@ -0,0 +1,27 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+# Matchmaking rule formatted as a JSON string.
+# Comments are not allowed in JSON, but most elements support a description field.
+# For instructions on designing Matchmaking rule sets, please check:
+# https://docs.aws.amazon.com/gamelift/latest/flexmatchguide/match-design-ruleset.html
+RULE_SET_BODY = '{"ruleLanguageVersion":"1.0","teams":[{"name":"Players","maxPlayers":4,"minPlayers":2}]}'

```

Comment:
Don't we want a basic 2 player game? minPlayers should be 2 or with a rule to relax to 1?

Reply:
Sorry but I don't quite get that. minPlayers is already set to be 2 in the rule and maxPlayers is a required attributes. This is the same rule as we used in LY 1.x testing.